lib/foreman_maintain/concerns/logger.rb

```diff
@@ -1,13 +0,0 @@
-require 'logger'
-
-module ForemanMaintain
-  module Concerns
-    module Logger
-      def logger
-        @logger ||= ::Logger.new($stderr).tap do |logger|
```

Comment:
@iNecas ,

Yes, we can keep it.
I have removed it only because every time we need to include this concerns/logger.rb module.

Other way is instead of keeping lib/foreman_maintain/concerns/logger.rb file, we could rename 
lib/foreman_maintain/logger.rb file to lib/foreman_maintain/fm_logger.rb and use this as you mentioned.
So that it will not create conflicts with original Logger class.

What do you think? 
Please let me know your views then I will go with further changes.